### PR TITLE
Twitter.(major).Rebranding from Twitter to X

### PR DESCRIPTION
**Pull request by sayam-nasir**
Name of the connector changed from Twitter -> X
Icon changed from Twitter -> X
Description updated

### CODE REVIEW

**Review by vtalas — CHANGES_REQUESTED** [review]
@sayam-nasir I think it's not necessary moving Twitter to X, causing the breaking change. All we need to do is change the labels and descriptions in service.json and update icons

**Comment by sayam-nasir** [comment]
@vtalas alright, makes sense. I have minimized the changes now

File: src/appmixer/twitter/bundle.json [hunk 1]
```diff
@@ -1,12 +1,15 @@
 {
     "name": "appmixer.twitter",
-    "version": "2.0.2",
+    "version": "3.0.0",
```
Comment (vtalas) [inline src/appmixer/twitter/bundle.json:3]: it's not a breaking change now, right?

Comment (sayam-nasir) [reply src/appmixer/twitter/bundle.json:3]: I think it shouldn't be 
 Only icons and activity descriptions and label of app was changed

**Review by vtalas — APPROVED** [review]

**Pull request merged**